locallib.php

```diff
@@ -218,6 +221,16 @@ function accredible_quiz_submission_handler($event) {
         foreach ($accrediblecertificaterecords as $record) {
             // Check for the existence of an activity instance and an auto-issue rule.
             if ( $record and ($record->finalquiz or $record->completionactivities) ) {
+                // Load user grade to attach in the credential.
+                $data = new stdClass();
+                $data->includegradeattribute = $record->includegradeattribute;
+                $data->gradeattributegradeitemid = $record->gradeattributegradeitemid;
+                $data->gradeattributekeyname = $record->gradeattributekeyname;
+                $data->course = $record->course;
+
+                $gradeattributes = $usersclient->get_user_grades($data, $user->id);
```

Comment:
I think we can directly pass `$record` to `get_user_grades`.

Reply:
You are right, it was done like this as previously we were not sending the user id as a different parameter. I have updated this.